packages/cluster-ui/src/store/nodes/nodes.selectors.ts

```diff
@@ -29,3 +31,24 @@ export const nodeDisplayNameByIDSelector = createSelector(
     return result;
   },
 );
+
+export function getRegionFromLocality(locality: ILocality): string {
+  for (let i = 0; i < locality.tiers.length; i++) {
+    if (locality.tiers[i].key === "region") return locality.tiers[i].value;
+  }
+  return "";
```

Comment:
When would there be a case where this information is not available?

Reply:
when the locality is not setup, for example. Even during the setup some queries might run without the value yet.
Also, we always want to function to have a backup value to return in case something is wrong.